internal/repo/datasource/dummy.go

```diff
@@ -1,32 +1,33 @@
 package datasource
 
-import (
-	"github.com/PereRohit/util/log"
-
-	"github.com/vatsal278/html-pdf-service/internal/config"
-	"github.com/vatsal278/html-pdf-service/internal/model"
-)
-
-type dummyDs struct {
-	dummySvc *config.DummyInternalSvc
-}
-
-func NewDummyDs(dummySvc *config.DummyInternalSvc) DataSource {
-	return &dummyDs{
-		dummySvc: dummySvc,
-	}
-}
-
-func (d dummyDs) Ping(req *model.PingDs) (*model.DsResponse, error) {
-	// internal logic to access datasource, etc.
-	log.Info(d.dummySvc.Data)
-	return &model.DsResponse{
-		Data: req.Data,
-	}, nil
-}
-
-func (d dummyDs) HealthCheck() bool {
-	// actual check for service condition
-	log.Debug(d.dummySvc.Data)
-	return true
-}
+//
+//import (
```

Comment:
Remove this file if it's not required. Also remove the corresponding _test.go file.

Reply:
done